packages/react/src/components/TimePicker/TimePicker.tsx

```diff
@@ -41,23 +42,31 @@ export const TimePicker: React.FunctionComponent<ITimePickerProps> = ({
   useHour12 = false,
   timeRange,
   strings = getDefaultStrings(useHour12, showSeconds),
-  defaultValue,
-  onChange,
+  currentDate,
```

Comment:
Changing `defaultValue` to `currentDate` and `onChange` to `onTimeChange` is a breaking change. Since Fluent follows semver we cannot make this change.

If you want to introduce these APIs we need to deprecate the existing ones and maintain their functionality in addition to adding the new APIs.

Since we need to keep the `defaultValue` API I think it's worth looking at updating this control to support both controlled and uncontrolled usage. What I'm thinking:

1. Maintain `defaultValue` for uncontrolled usage as this is the React idiom.
2. Introduce a `value` prop for controlled usage.

The [`useControllableValue`](https://github.com/microsoft/fluentui/blob/master/packages/react-hooks/src/useControllableValue.ts) hook will help with this.

Reply:
Resolved by reverting all props to previous names (switched back to `defaultValue` and `onChange`), and then added a new `value` prop for handling controlled usage.